common/utils/timezones.js

```diff
@@ -0,0 +1,25 @@
+export const TIMEZONES = [
+  { name: "Europe/Paris", label: "France Métropolitaine" },
+  { name: "America/Guadeloupe", label: "Guadeloupe" },
+  { name: "America/Martinique", label: "Martinique" },
+  { name: "America/Guyana", label: "Guyane" },
```

Comment:
:red_circle: La timezone qui correspond à la Guyane Française est *America/Cayenne* . 
```suggestion
  { name: "America/Cayenne", label: "Guyane" },
```